Description: Patch to if_ and if_err plugins to detect msh|venet|veth
 interfaces.
Origin: other, http://bugs.debian.org/489505
Bug-Debian: http://bugs.debian.org/489505
Forwarded: no
Index: munin-1.2.6/node/node.d.linux/if_.in
===================================================================
--- munin-1.2.6.orig/node/node.d.linux/if_.in	2008-07-20 21:32:46.000000000 +0200
+++ munin-1.2.6/node/node.d.linux/if_.in	2008-07-20 22:04:15.000000000 +0200
@@ -93,7 +93,7 @@
 
 if [ "$1" = "suggest" ]; then
 	if [ -r /proc/net/dev ]; then
-		egrep '^ *(eth|wlan|ath|ra)[0-9]+:' /proc/net/dev | cut -f1 -d: | sed 's/ //g'
+		egrep '^ *(eth|wlan|ath|ra|msh|venet|veth)[0-9]+:' /proc/net/dev | cut -f1 -d: | sed 's/ //g'
 		exit 0
 	else
 		exit 1
